CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+  - [How do I send in an agreement?](#how-do-i-send-in-an-agreement)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Submitting bugs
+
+If you find a security vulnerability, do NOT open an issue. Email
+security@artefactual.com instead.
+
+Issues should be filed in our collective
+[Archivematica related issues repository][am-issues]
+
+For user support, clarification, or issues related to your deployment and use
+of AMClient you can post in our [Archivematica forum][am-forum]. A post to the
+mailing list is always welcome!
+
+Useful questions to answer if you're having problems include:
+
+* What version of AMClient are you using?
+* How was it installed?
+* How was AMClient being used when this issue occurred?
+* What was the expected outcome?
+* What was the actual outcome?
+* Can we reproduce this reliably?
+
+## Contribution standards
+
+For more information on contribution guidelines and standards, see the
+CONTRIBUTING.md in the [Archivematica project][contribute-to-am].
+
+## Contributor's Agreement
+
+In order for the Archivematica development team to accept any patches or code
+commits, contributors must first sign this
+[Contributor's Agreement][contibutors-agreement].
+
+The Archivematica contributor's agreement is based almost verbatim on the
+[Apache Foundation][apache-org]'s individual
+[contributor license][contirbutor-license].
+
+If you have any questions or concerns about the Contributor's Agreement, please
+email us at agreement@artefactual.com to discuss them.
+
+### Why do I have to sign a Contributor's Agreement?
+
+One of the key challenges for open source software is to support a
+collaborative development environment while protecting the rights of
+contributors and users over the long-term. Unifying Archivematica copyrights
+through contributor agreements is the best way to protect the availability and
+sustainability of Archivematica over the long-term as free and open-source
+software. In all cases, contributors who sign the Contributor's Agreement
+retain full rights to use their original contributions for any other purpose
+outside of Archivematica, while enabling Artefactual Systems, or any successor
+organization which may eventually take over responsibility for Archivematica,
+and the wider Archivematica community, to benefit from their collaboration and
+contributions in this open source project.
+
+[Artefactual Systems][af-systems] has made the decision and has a proven track
+record of making our intellectual property available to the community at large.
+By standardizing contributions on these agreements the Archivematica
+intellectual property position does not become too complicated. This ensures
+our resources are devoted to making our project the best they can be, rather
+than fighting legal battles over contributions.
+
+### How do I send in an agreement?
+
+Please print out, read, sign, and scan the
+[contributor agreement][contibutors-agreement] and email it to
```

Comment:
```suggestion
[contributor agreement][contributors-agreement] and email it to
```

Reply:
Ha! Typical I fixed the path but forgot the docs! Good spot!